cmd/livepeer/livepeer.go

```diff
@@ -181,8 +175,34 @@ func main() {
 		}
 	}
 
+	var gethCmd *exec.Cmd
 	//Set up ethereum-related stuff
-	if *gethipc != "" {
+	if *ethDatadir != "" {
+		gethipc := filepath.Join(filepath.Join(*ethDatadir, "geth.ipc"))
+		//If getipc file is not there, start the geth node
+		if _, err := os.Stat(gethipc); os.IsNotExist(err) {
+			//Set up geth params depending on network, invoke geth
+			if *testnet {
+				gethCmd = exec.Command("geth", "--rpc", "--datadir", *ethDatadir, "--bootnodes=enode://a1bd18a737acef008f94857654cfb2470124d1dc826b6248cea0331a7ca82b36d2389566e3aa0a1bc9a5c3c34a61f47601a6cff5279d829fcc60cb632ee88bad@13.58.149.151:30303") //timeout in 3 mins
```

Comment:
so here we hard code the geth bootnode, but above we have a flag for the livepeer bootnode? I could see this getting a little confusing.

Reply:
I agree.  Will make changes here